tools/codegen/Cargo.toml

```diff
@@ -30,7 +30,7 @@ serde_json = { version = "1.0.61", optional = true }
 serde_yaml = { version = "0.8.15", optional = true }
 
 [dev-dependencies]
-tempfile = "3"
+tempfile = "=3.6.0"
```

Comment:
Latest `tempfile` need rust-toolchain be `1.63.0`, but molecule's MSRV provide `1.56.1`, so I have to specify `tempfile`'s version to `3.6.0`: https://crates.io/crates/tempfile/versions